src/modalities/models/components/layer_norms.py

```diff
@@ -0,0 +1,92 @@
+from typing import Annotated
+
+import torch
+import torch.nn as nn
+from pydantic import BaseModel, Field
+from torch.nn import functional as F
+
+from modalities.config.lookup_enum import LookupEnum
+
+
+class LayerNormIF(nn.Module):
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        raise NotImplementedError
+
+    def __copy__(self):
+        raise NotImplementedError
+
+
+class RMSLayerNorm(LayerNormIF):
+    def __init__(self, ndim: int, epsilon: float = 1e-6):
```

Comment:
Should we not implement an (optional) bias for `RMSLayerNorm`, just like we do for `ZLayerNorm`? The original RMSNorm paper uses a bias by default.

Reply:
Good point! I also checked the original RMSNorm implementation and they also had it (see: https://github.com/bzhangGo/rmsnorm/blob/2e726f1a3f106bb719056422f4f9b6aca03d3ce6/rmsnorm_torch.py#L32)